config/initializers/inflections.rb

```diff
@@ -13,4 +13,5 @@
 # These inflection rules are supported but not enabled by default:
 ActiveSupport::Inflector.inflections(:en) do |inflect|
   inflect.acronym 'SLA'
+  inflect.irregular "analytics", "analytics"
```

Comment:
should one of these be "analytic"?

Reply:
whats happening here is that im overriding the analytics singular to also be analytices, because the model name should indicate one SET of analytics.